commands/auth/login/login.go

```diff
@@ -255,8 +256,9 @@ func hostnameValidator(v interface{}) error {
 	if len(strings.TrimSpace(val)) < 1 {
 		return errors.New("a value is required")
 	}
-	if strings.ContainsRune(val, '/') || strings.ContainsRune(val, ':') {
-		return errors.New("invalid hostname")
+	re := regexp.MustCompile("^[A-Za-z0-9][A-Za-z0-9-\\.]*$")
```

Comment:
> should use raw string (\`...\`) with regexp.MustCompile to avoid having to escape twice (gosimple)

Also the escape sequence `\.` is redundant

```suggestion
	re := regexp.MustCompile(`^[A-Za-z0-9][A-Za-z0-9-.]*$`)
```